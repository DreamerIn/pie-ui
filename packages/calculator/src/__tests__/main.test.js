import React from 'react';
import Main from '../main';

test('render a calculator component', () => {
  const model = { mode: 'basic' };
  const wrapper = shallow(
    <Main model={model} />
  );
  expect(wrapper).toMatchSnapshot();
});