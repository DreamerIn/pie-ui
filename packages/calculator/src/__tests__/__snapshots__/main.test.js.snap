// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a calculator component 1`] = `
<Main
  classes={
    Object {
      "icon": "Main-icon-1",
    }
  }
  model={
    Object {
      "mode": "basic",
    }
  }
/>
`;
